DocAI Incubator Tools/best-practices/Child_Entity_Tag_Using_Header_Keyword.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<p>Maybe rephrase this since they don't need to copy/paste in a notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>